wagtail/contrib/modeladmin/templates/modeladmin/includes/search_form.html

```diff
@@ -6,5 +6,8 @@
             <input id="id_q" name="{{ search_var }}" value="{{ view.query }}" placeholder="{% blocktrans trimmed with view.verbose_name_plural|lower as name %}Search {{ name }}{% endblocktrans %}" type="text">
         {% endfield %}
         <button type="submit" class="w-sr-only">{% trans 'Search' %}</button>
+        {% for pair in view.params.items %}
```

Comment:
Nice. Can we add a simple comment here maybe to explain this - might be unclear to future developers. 

Also should we unpack the pair key/values to clearer names. 

Code seems good just trying to help for future ability to grok. 

I'll test this locally - haven't done that yet.

Reply:
@lb- Good points! What do you think of this:
```html
{# Keep all parameters from the query string (e.g. filter state). #}
{% for name, value in view.params.items %}
    {% if name != search_var %}<input type="hidden" name="{{ name }}" value="{{ value }}">{% endif %}
{% endfor %}
```

Reply:
love it